ibm/mas_devops/playbooks/oneclick_core.yml

```diff
@@ -34,25 +34,13 @@
         fail_msg: "One or more required environment variables are not defined"
 
     - name: Check for required environment variables for UDS
-      when: (bas_provider =='UDS')
       assert:
```

Comment:
This changes means the user has to set the UDS_X variables even if this is a new user that wants DRO and doesn't know what UDS is. I think we should put back this when condition